### ISSUE TITLE
Script does not recognize World Tour game folder

### ISSUE THREAD

**Issue by 333hronos** [NightFright2k19/worldtour_raze#5]
Hello, I tried press Enter, and also manually enter
"F:/SteamLibrary/steamapps/common/Duke Nukem 3D Twentieth Anniversary World Tour" (with and without quotes, with slash and back slash, and remove part of the path down to root)

Script does not recognize World Tour game folder anyway.

**Comment by NightFright2k19** [comment]
Is the script aborted with an error or does it get stuck? Also mind the Win10 requirement.

**Comment by 333hronos** [comment]
Yes, it's aborted with error "game folder not found..." 

I found workaround: I create symbolic link for my game in C / Program Files / Steam / Common folder,  as the script expect to.

Maybe someone find it useful.

p.s. Yes, I have Win10.
p.p.s. Also I found, that Raze engine port natively support Duke Nukem episodes 1-4, and the new 5th.

**Comment by NightFright2k19** [comment]
Is Steam even installed? Without Steam registry keys, the script would ofc not find the game folder. (However, it would not explain why manually entering the directory won't work. The correct folder to choose should be _F:\SteamLibrary_ in your case. The rest, i.e. _steamapps\common\Duke Nukem 3D Twentieth Anniversary World Tour_, is added by the script.)

**Comment by 333hronos** [comment]
Yes, Steam is installed, and I also manually checked two registry keys, that mentioned in the script code as requirements.

**Comment by NightFright2k19** [comment]
Check if this test file runs through without issues. It has its abort conditions adjusted. (Rename file ending to .bat before, ofc.)

[worldorder_test.txt](https://github.com/user-attachments/files/18286737/worldorder_test.txt)

**Comment by 333hronos** [comment]
> Check if this test file runs through without issues. It has its abort conditions adjusted. (Rename file ending to .bat before, ofc.)
> 
> [worldorder_test.txt](https://github.com/user-attachments/files/18286737/worldorder_test.txt)

This script only blinks on half of second and that's all
![image](https://github.com/user-attachments/assets/d46d514f-539d-4701-8c67-f20d4de2953c)

**Comment by NightFright2k19** [comment]
No development since months, closing.
